sagemaker/run/gconstruct_entry.py

```diff
@@ -32,12 +32,15 @@ def parse_construct_args():
         required=True, help="Path to store output")
     parser.add_argument("--graph-name", type=str,
         required=True, help="Graph name")
-    return parser
+
+    return parser.parse_known_args()
 
 
 if __name__ =='__main__':
-    parser = parse_construct_args()
-    args, unknownargs = parser.parse_known_args()
+    args, unknownargs = parse_construct_args()
```

Comment:
Why change the code?

Reply:
The intention is to match function outcome to its name, `parse_construct_args` implies the function parses arguments, but previously it was returning a parser. In addition, the `parser` var in the function was shadowing the `parser` var inside the main function so pylance was reporting a warning.